### ISSUE TITLE
Update to HttpClient

### ISSUE THREAD

**Issue by dsoltesz** [Breeze/breeze.bridge.angular#17]
can we have the bridge updated to use the new httpClient instead of http.

**Comment by martindalec** [comment]
Any movement on this? Without it using HttpClient, we can't intercept the requests to append any type of authorization headers native to Angular 4

**Comment by marcelgood** [comment]
It's on the list. Just need to set aside some time. Currently haven't needed it on any of our own projects, so it's remained a lower priority.

**Comment by dsoltesz** [comment]
Just following up on this especially since angular 5 is out

**Comment by marcelgood** [comment]
Sorry about the continued delay. I'm working on this now, however making it work with Angular 5 apparently isn't as straight forward as I hoped. I have to redo the library to make it consumable in Angular 5. Hopefully won't take too long. I'm aiming for next week, but I'm waiting on a dependency that should be released this weekend before I can build the library.

**Comment by marcelgood** [comment]
The beta for the new package supporting HttpClient has been released. 

https://github.com/Breeze/breeze.bridge2.angular
